.env

```diff
@@ -1,3 +1,4 @@
 # Docker environment
 DATA_STORE_DB_USERNAME=ccd
 DATA_STORE_DB_PASSWORD=ccd
+SERVER_PORT=4452
```

Comment:
I assume this cleanup isn't directly needed for the swagger docs? `SERVER_PORT` does make me wonder if we can use this in Dockerfiles... he thinks aloud.

Reply:
it's needed in the way that the publish script needs a `SERVER_PORT` env defined and achieves that by sourcing the `.env` file.